src/store/store.ts

```diff
@@ -1116,8 +1122,9 @@ export const useStore = defineStore('store', {
     setCurrentJsDate (date: Date) {
       this.stateModel.currentJsDate = date
     },
-    setContinuationInBusinessInfo (val: any) {
+    setExistingBusinessInfo (val: ExistingBusinessInfoIF) {
       this.stateModel.continuationIn.existingBusinessInfo = val
+      if (!this.stateModel.ignoreChanges) this.stateModel.haveChanges = true
```

Comment:
This sets `haveChanges` if the user has done something to set this data object. This flag is used to prompt the user if they try to leave this web app before saving their changes.

The "if" on line 1127 is to skip this line conditionally -- when the app starts (see mounted() in various steps).